dcos_tunnel/cli.py

```diff
@@ -401,6 +413,8 @@ def _socks(port, config_file, user, privileged, ssh_port, host, option):
     host = get_host(host)
 
     scom = "ssh -N -D {} {} {}@{}".format(port, ssh_options, user, host)
+    if verbose:
+        scom += ' -v'
```

Comment:
can you add this as a format field instead, like setting it to a empty string and populating it if verbose is set, and add another `{}` to the format in `scom`